nodeJS/orms/prisma_orm.md

```diff
@@ -0,0 +1,157 @@
+### Introduction
+
+Tired of writing raw SQL queries in your projects? In this lesson, we'll talk about Object Relational Mappers (ORM for short). ORMs are tools that let you manipulate data in your database and are widely used in the software industry. We'll dive deep into one ORM popular in the Node.js landscape: [Prisma ORM](https://www.prisma.io/).
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What are ORMs, and why are they used?
+- Introduction to Prisma ORM
+- Features of Prisma
```

Comment:
For consistency's sake, my thoughts are this should be kept as either Prisma or Prisma ORM throughout the lesson rather than bouncing between the two which could imply they are different. [The official (full) name is Prisma ORM](https://www.prisma.io/docs/orm/overview/prisma-in-your-stack/is-prisma-an-orm), if that's what we want to go with.

Reply:
Agreed.